gc/base/GCExtensionsBase.hpp

```diff
@@ -267,6 +267,7 @@ class MM_GCExtensionsBase : public MM_BaseVirtual {
 	uintptr_t oldHeapSizeOnLastGlobalGC;
 	uintptr_t freeOldHeapSizeOnLastGlobalGC;
 	float concurrentKickoffTenuringHeadroom; /**< percentage of free memory remaining in tenure heap. Used in conjunction with free memory to determine concurrent mark kickoff */
+	float tenureSOABytesDeviationBoost;
```

Comment:
add a comment somethink like: boost factor for tenuring deviation used for concurrent mark kickoff math